Apex and SOQL doc Update (#4558)
forcedotcom/salesforcedx-vscode: docs/_articles/en/apex/apex-tasks.md

---
title: Apex Tasks
lang: en
---

> This repository contains sample Apex tasks that you can complete in Code Builder or VS Code Extensions for desktop. We constantly add to this list, so make sure you visit this topic often. 

Reviewer: ```suggestion
This repository contains sample Apex tasks that you can complete in Code Builder or VS Code using the Salesforce Extensions. We constantly add to this list, so make sure you visit this topic often. 
```

Reviewer: If we're good with this change then it'll apply across the other intros as well

> This is a repository of sample tasks using Apex that you can complete in Code Builder or VS Code Extensions for desktop. We’ll keep adding to this list, so make sure you visit this topic often. 

Reviewer 2: ```suggestion
This repository contains sample Apex tasks that you can complete in Code Builder or VS Code Extensions for desktop. We constantly add to this list, so make sure you visit this topic often. 
```

## Use Anonymous Apex to Mass Update Account Ratings

> Say you have accounts with `Rating` set to `Cold` because of unfavorable business conditions. You now want to make mass updates to these accounts and change their rating to `Warm`.  Say you want to limit this update to mid-sized accounts with more than a certain amount of revenue. 

Reviewer 2: ```suggestion
Say you have accounts with `Rating` set to `Cold` because of unfavorable business conditions. You now want to make mass updates to these accounts and change their rating to `Warm`.  You want to limit this update to mid-sized accounts with more than a certain amount of revenue. 
```

Reviewer 2: I think two sentences staring with "say" is overkill -- the first one establishes that this is the hypothetical use case.

> This Anonymous Apex code updates ratings of accounts that have an annual revenue greater than `$10,000,000` and have more than `1000` employees:

Reviewer 2: ```suggestion
This anonymous Apex code updates ratings of accounts that have an annual revenue greater than `$10,000,000` and have more than `1000` employees:
```

Reviewer 2: See https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_anonymous_block.htm in which we don't capitalize "anonymous" when used within a sentence.  Might as well keep it consistent :)

Author: We've gone back and forth about this. I'll make updates in other topics also.

Reviewer 2: Also, can you reorg so that the "This anonymous Apex code..." sentence comes right before the code?  Right now it's coming before the numbered list, so the colon looks odd.  Maybe you can keep the sentence where it is and just say:

The anonymous Apex code updates ratings of accounts that have an annual revenue greater than `$10,000,000` and have more than `1000` employees.

Or move the sentence to right _below_ the code.

> 1. In the `scripts/apex` folder create a new Anonymous Apex file using the `.apex` file extension.

Reviewer 2: ```suggestion
1. In the `scripts/apex` folder, create an anonymous Apex file using the `.apex` file extension.
```

> Paste the code into the file:

Reviewer 2: ```suggestion
Paste this code into the file:
```
